### DISCUSSION TITLE
Versioning - keep X days grayed out

### DISCUSSION THREAD

**Discussion by michaelclaesson** [pimcore/pimcore#9363] [Q&A / Support] [answered]
Hi!

I've noticed a peculiar thing and that is that the setting for versioning of documents, assets and objects for X amount of days always appear as greyed out. Why is that? It would be really neat to keep versions on that basis instead of a specific amount of versions. Sometimes a person saves a lot of versions during the span of just a day or two and in that case keeping versions for let's say 30 or 60 days would be nice. 

Appreciate any advice on this! Thanks!

**Comment by aweichler** [comment, marked as answer]
Hi,

when you clear the value from "Store version history for x Steps", you should be able to change the value for "Store version history for x Days".

**Reply by michaelclaesson** [reply]
So it was that easy! Thanks a bunch! 🙇🏻‍♂️
